### ISSUE TITLE
VaStepper keyboard navigation and aria attributes

### ISSUE THREAD

**Issue by m0ksem** [epicmaxco/vuestic-ui#3028]
The component should be accessible: keyboard navigation + aria attributes.

https://docs.telerik.com/kendo-ui/controls/navigation/stepper/accessibility/keyboard-navigation
https://stackoverflow.com/questions/52932018/making-a-step-progress-indicator-accessible-for-screen-readers

_Originally posted by @aluarius in https://github.com/epicmaxco/vuestic-ui/pull/2906#pullrequestreview-1302871577_
